lib/AST/Substitution.cpp

```diff
@@ -63,7 +70,8 @@ Substitution Substitution::subst(Module *module,
     // If we have a concrete conformance, we need to substitute the
     // conformance to apply to the new type.
     if (c.isConcrete()) {
-      auto substC = c.getConcrete()->subst(module, substReplacement, subMap);
+      auto substC = c.getConcrete()->subst(module, substReplacement,
```

Comment:
It's also used here: https://github.com/jckarter/swift/blob/6fa0c97e081f319c1962061269ad29fb76255093/lib/AST/ProtocolConformance.cpp#L483